site/content/contribute/server/developer-setup/arch.md

```diff
@@ -20,52 +20,54 @@
     psql -U mmuser -W -h 127.0.0.1 -f mm_postgres_backup.bak mattermost_test
     ```
     If you don't migrate your data, the new, docker-compose-managed containers will start out empty. To remove the old containers -- destroying any existing data -- use `make clean-old-docker`.
-    
+
 2. Install docker-compose
 
     ```sh
     sudo pacman -S docker-compose
     ```
 
 
-4. Install Go:
+3. Install Go:
 
     ```sh
     sudo pacman -S base-devel
     sudo pacman -S go
     ```
 
-5. Update your shell's initialization script (e.g. `.bashrc` or `.zshrc`) and add the following:
+4. Update your shell's initialization script (e.g. `.bashrc` or `.zshrc`) and add the following:
 
     ```sh
     export GOPATH=$HOME/go
     export PATH=$PATH:$GOPATH/bin
     export PATH=$PATH:/usr/local/go/bin
     ```
 
-6. Edit `/etc/security/limits.conf` as an administrator (e.g. `sudo`) and add the following lines, replacing `{username}` with your username:
+5. Edit `/etc/security/limits.conf` as an administrator (e.g. `sudo`) and add the following lines, replacing `{username}` with your username:
 
     ```sh
     {username}  soft  nofile  8096
     {username}  hard  nofile  8096
     ```
 
-7. Logout and login to effect the changes above.
+6. Logout and login to effect the changes above.
 
-8. Fork https://github.com/mattermost/mattermost-server
+7. Fork https://github.com/mattermost/mattermost-server
 
-9. Clone the Mattermost source code from your fork:
+8. Clone the Mattermost source code from your fork:
 
     ```sh
-    export GITHUB_USERNAME=my_username
-    mkdir -p $(go env GOPATH)/src/github.com/mattermost
-    git clone https://github.com/$GITHUB_USERNAME/mattermost-server.git $(go env GOPATH)/src/github.com/mattermost/mattermost-server
+    git clone https://github.com/YOUR_GITHUB_USERNAME/mattermost-server.git
+    cd mattermost-server
+    git config core.eol lf
+    git config core.autocrlf input
```

Comment:
I don't think we should have these here. Also, why do reset on fresh clone? 🤔

Reply:
@reflog do you mean only this line? or both `git config core.*` lines?

Reply:
lines 61-64

Reply:
done